README.md

```diff
@@ -35,14 +35,14 @@ The Operator’s job is to:
 
 ### Downloader ###
 
-Stahuje obsah a metadata nezpracovaných URL adres.
+The Downloader fetches and saves the contents and metadata from the unprocessed URL addresses.
 
-Činnost _downloaderu_ se skládá z:
+The Downloader’s job is to:
 
-1. načítání URL ke stažení,
-2. kontroly stažených URL,
-3. stahování obsahu URL a případných metadat,
-4. uložení stažených dat.
+1. read the list of URLs for download,
```

Comment:
Hmmm. You are correct that reading the URLs from the file is an implementation detail, although there needs to be a persistent representation of the list. The actual passing of the URLs can happen through a file, a queue, an interprocess communication, etc.

I would, however, keep the word “read” as it corresponds to the original “načítání.” “Receiving” is something different, and the scope of this pull request is only to translate the original version; it should not introduce changes to the meaning.

Reply:
I simplified the sentence so it does not mention a list. I kept the plural because it seemed more agnostic to me. Even if done one by one, the Downloader still reads multiple URLs.